index.bs

```diff
@@ -2643,15 +2629,27 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
    |command parameters| if present, or null otherwise.
 
 1. Let |max depth| be the value of the <code>maxDepth</code> field of |command
-   parameters| if present, or 0 otherwise.
+   parameters| if present, or null otherwise.
 
 1. Let |depth| be 0.
```

Comment:
Is |depth|  still used in the current version?

Reply:
No, it's a left-over and I'll remove. 

Note that this follows what we have for serialization and if we want to do the rename I would propose a different issue / PR.